quickwit/quickwit-control-plane/src/indexing_scheduler/scheduling/README.md

```diff
@@ -0,0 +1,105 @@
+# Scheduling logic
+
+Quickwit needs to assign indexing tasks to a set of indexers nodes.
+We call the result of this decision the indexing physical plan.
+
+This needs to be done under the constraints of:
+- not exceeding the maximum load of each node. (O)
+
+We also want to observe some interesting properties such as:
+- (A) we want to keep the current state of the cluster.
```

Comment:
What do you mean by "keep the current state of the cluster"? Do you mean "healthy cluster"?

Reply:
Ok, I understood with the distance between two solution matrices. You could add this sentence to clarify: "we don't want to move indexing tasks between indexers."